### ISSUE TITLE
Call to undefined method Google\Auth\HttpHandler\Guzzle5HttpHandler::buildPsr7Response()

### ISSUE THREAD

**Issue by gustavomanolo** [google/google-api-php-client#864]
HI, i'm trying to upload a file to a bucket but i'm getting the following error:
Call to undefined method Google\Auth\HttpHandler\Guzzle5HttpHandler::buildPsr7Response()

```php
	$client = new Google_Client();
	$client->setApplicationName("myapp");


	$client->setAuthConfig( 'config.json' );


	$storage = new Google_Service_Storage( $client );

	/***
	 * Write file to Google Storage
	 */
	try 
	{
		$file_name = 'newfile.jpg';

		$obj = new Google_Service_Storage_StorageObject();
		$obj->setName( $file_name );
		

		$storage->objects->insert(
			"timbrao",
			$obj,
			['name' => $file_name, 'data' => file_get_contents("file.jpg"), 'uploadType' => 'media']
		);
		
	}      
	catch (Exception $e)
	{
		print $e->getMessage();
	}
```

**Comment by bshaffer** [comment]
apparently #808 was submitted with a stubbed method, and `buildPsr7Response` doesn't actually exist... That's bad git karma.

Thanks for finding this.

**Comment by bshaffer** [comment]
fixed in #864
